lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
+  <h2>${_("Entrance Exam grade adjustment")}</h2>
+  <div class="request-response-error"></div>
+  <p>
+  ${_("Specify the {platform_name} email address or username of a student here:").format(platform_name=settings.PLATFORM_NAME)}
+  <input type="text" name="entrance-exam-student-select-grade" placeholder="${_("Student Email or Username")}">
+  </p>
+  <br>
```

Comment:
We should not be styling with paragraph and break tags. IMO we should be using some of the "form-field" classes. @frrrances, who would you recommend to do a UI review of this PR?